### PR TITLE
dark mode

### LINKED ISSUE #54: Dark mode is not 100% supported yet
Screen Shot 2023-01-18 at 12 37 40

### PR DESCRIPTION
**Pull request by nguyentranchung**
fix #54

### CODE REVIEW

**Comment by nguyentranchung** [comment]
Screen Shot 2023-01-18 at 12 44 05

**Comment by volkar** [comment]
Definitely need this to be merged 👍

**Comment by mohamedsabil83** [comment]
Thanks for contributing @nguyentranchung. Does this PR need to merge (#53) first?

**Comment by volkar** [comment]
I uses this PR, it's independed and works perfect on current version of Tinymce that uses your repository.

File: resources/views/tiny-editor.blade.php [hunk 1]
```diff
@@ -22,8 +22,8 @@ class="relative z-0"
                     tinymce.init({
                         target: $refs.tinymce,
                         language: '{{ $getInterfaceLanguage() }}',
-                        skin: typeof theme != 'undefined' ? theme : 'light',
```
Comment (mohamedsabil83) [inline resources/views/tiny-editor.blade.php:25]: This line prevents an error established when **darkMode** is disabled. revert it

File: resources/views/tiny-editor.blade.php [hunk 2]
```diff
@@ -22,8 +22,8 @@ class="relative z-0"
...
-                        content_css: this.skin === 'dark' ? 'dark' : 'default',
+                        skin: theme === 'dark' ? 'oxide-dark' : 'oxide',
```
Comment (mohamedsabil83) [inline resources/views/tiny-editor.blade.php:25]: Return **theme** in these two line to **this.skin**

**Comment by volkar** [comment]
It must be
```
skin: typeof theme != 'undefined' && theme === 'dark' ? 'oxide-dark' : 'oxide',
content_css: typeof theme != 'undefined' && theme === 'dark' ? 'dark' : 'default',
```
...to prevent undefined error then. I'll pull another request to fix body_class PR #57

File: resources/views/tiny-editor.blade.php [hunk 3]
```diff
@@ -22,8 +22,8 @@ class="relative z-0"
...
+                        content_css: theme === 'dark' ? 'dark' : 'default',
```
Comment (mohamedsabil83) [inline resources/views/tiny-editor.blade.php:26]: Revert **theme** in this line to **this.skin**

**Comment by mohamedsabil83** [comment]
> It must be
> 
> ```
> skin: typeof theme !== 'undefined' && theme === 'dark' ? 'oxide-dark' : 'oxide',
> content_css: typeof theme !== 'undefined' && theme === 'dark' ? 'dark' : 'default',
> ```
> 
> ...to prevent undefined error then. I'll pull another request to fix body_class PR #57

Just update the review. I meant line **26**

**Comment by mohamedsabil83** [comment]
Done in PR #58

**Comment by nguyentranchung** [comment]
thanks <3

**Comment by mohamedsabil83** [comment]
You're welcome

**Pull request closed without merging**